announcement/announcement-api/api/src/bundle/announcement.properties

```diff
@@ -196,6 +196,8 @@ revise.required.star = Required items marked with a <span class="reqStarInline">
 revise.update = Update the form, then choose the appropriate button at the bottom.
 revise.announce = Announcement
 revise.announcement.body = Body
+revise.highlight = <strong>Highlight this announcement.</strong> It will be more prominent in the announcement list.
```

Comment:
Please use two strings and do not put markup in translation strings, markup must be in the VM file.

Reply:
Alright. I will do the changes

Reply:
Forget it, this file is full of this bad patterns.